api/envoy/config/metrics/v3/stats.proto

```diff
@@ -259,6 +262,47 @@ message TagSpecifier {
   }
 }
 
+// Specifies a matcher for stats and the buckets that matching stats should use.
+message HistogramBucketSettings {
+  // The stats that this rule applies to.
+  type.matcher.v3.StringMatcher match = 1 [(validate.rules).message = {required: true}];
```

Comment:
How does this interact with tags? Is it the name including tags? Seems like you usually want to ignore the things that get extracted as tags and match directly on the kind of metric, but maybe there are some situations where you would want, for the same stat, different behavior on listener A vs. B.

Reply:
It operates on the name before tag-extraction, ie the statsd name. It should be easy enough to ignore tags with a regex, or for some things, just do a suffix-match.